### PR TITLE
Fetch 32 featured apps instead of default 10

### PR DESCRIPTION
**Pull request by faymontage**

### CODE REVIEW

File: Token/APIClients/AppsAPIClient.swift [hunk 1]
```diff
@@ -30,7 +30,7 @@ public class AppsAPIClient: NSObject, CacheExpiryDefault {
     }
 
     func getFeaturedApps(completion: @escaping (_ apps: [TokenUser], _ error: Error?) -> Void) {
-        self.teapot.get("/v1/apps/featured") { (result: NetworkResult) in
+        self.teapot.get("/v1/apps/featured?limit=32") { (result: NetworkResult) in
```
Comment (roberthein) [inline Token/APIClients/AppsAPIClient.swift]: Before on Slack you mentioned that this should be a multiple of 3. That's not the case anymore?

Comment (faymontage) [inline Token/APIClients/AppsAPIClient.swift]: I thought the rows were 3 items (was I dreaming?) - they seem to be 2.

Comment (Elland) [inline Token/APIClients/AppsAPIClient.swift]: I changed it to two a while ago, to give it more spacing and avoid clipping a lot of bot names. I think @roberthein will start today on the new browser view layout?

Comment (roberthein) [inline Token/APIClients/AppsAPIClient.swift]: My plan is to make the transition interactive first and after that process @Elland's feedback on the PR.

Comment (faymontage) [inline Token/APIClients/AppsAPIClient.swift]: For the purposes of this PR, 32 is the correct number yes.

**Pull request merged**